src/test/java/xjs/data/serialization/parser/CommonParserTest.java

```diff
@@ -41,6 +41,12 @@ public final void parse_readsDecimal() throws IOException {
         assertEquals(12.34, this.parse("12.34").asDouble());
     }
 
+    @Test
```

Comment:
This needs to go in `DjsParserTest` since it should not apply to JSON or other formats.